src/Jupyter/Jupyter.csproj

```diff
@@ -6,6 +6,7 @@
     <RootNamespace>Microsoft.Quantum.IQSharp.Jupyter</RootNamespace>
     <AssemblyName>Microsoft.Quantum.IQSharp.Jupyter</AssemblyName>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>1591</NoWarn>
```

Comment:
Should we open an issue to add all missing documentation for publicly-accessible elements and re-enable this warning?